### PR TITLE
dev: Update for k3d v5.1.0 | k3d-action@v2

### PR DESCRIPTION
**Pull request by somaritane**
- Updated GH worfkows to use AbsaOSS/k3d-action@v2
- Updated documentation for min k3d version (v5.1.0)
- Updated k3d config files for new schema (k3d.io/v1alpha3)

Signed-off-by: Timofey Ilinykh <ilinytim@gmail.com>

### CODE REVIEW

**Review by k0da — DISMISSED** [review]

File: Makefile [hunk 1]
```diff
@@ -225,7 +225,6 @@ destroy-full-local-setup: ## Destroy full local multicluster setup
 	@for c in $(CLUSTER_IDS); do \
 		k3d cluster delete $(CLUSTER_NAME)$$c ;\
 	done
-	docker network rm $(CLUSTER_GSLB_NETWORK)
```
Comment (kuritka) [inline Makefile:228]: remove `CLUSTER_GSLB_NETWORK` variable

**Review by kuritka — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (k0da) [reply Makefile:228]: @kuritka I'll need it later

Comment (kuritka) [reply Makefile:228]: @k0da , ok, than LGTM, thanks.

**Review by kuritka — APPROVED** [review]

**Comment by somaritane** [comment]
@k0da, @kuritka rebased branch from upstream, pls re-check

**Review by k0da — DISMISSED** [review]
LGTM

**Review by kuritka — DISMISSED** [review]

**Review by k0da — APPROVED** [review]

**Review by kuritka — APPROVED** [review]

**Pull request merged**